config.go

```diff
@@ -1,9 +1,7 @@
 package corbado
```

Comment:
can you add a "NewFromEnd()" method just like the PHP fromEnv() one?

Reply:
@corbadoman there was a `fromEnv` method but me and martin decided to remove it so developers are not forced to use the env variables names we set .